src/threading.c

```diff
@@ -92,41 +92,7 @@ JL_DLLEXPORT void jl_set_safe_restore(jl_jmp_buf *sr)
 // as the fallback in the shared object. For better efficiency, we also
 // create a `__thread` variable in the main executable using a static TLS
 // model.
-#if defined(_OS_DARWIN_)
```

Comment:
Perhaps also adjust the comments in line 85 and line 91, to mention macOS in the right place?